keyboards/kbdcraft/adam64/readme.md

```diff
@@ -0,0 +1,27 @@
+# KBDCraft Adam64
+
+![Adam64](https://i.imgur.com/6m2yyTUh.jpg)
+
+A custom 64-key mechanical keyboard comprised of Legos.
+
+* Keyboard Maintainer: The QMK Community
+* Hardware Supported: STM32F401
+* Hardware Availability: https://kbdcraft.store/products/adam
+
+Make example for this keyboard (after setting up your build environment):
+
+    make kbdcraft/adam64:vial
+
+Flashing example for this keyboard:
+
+    make kbdcraft/adam64:vial:flash
```

Comment:
I do see reason behind this, but looking through the repo, it seems are are some `readme.md`s that use `...:vial:flash` instead of `...:default:flash`, but that's just me.